pkg/install/install.go

```diff
@@ -528,13 +534,12 @@ func (o *Install) runInstallWizard() error {
 	if err := survey.AskOne(
 		pOps,
 		&opIndexes,
-		survey.WithValidator(survey.MinItems(1)),
```

Comment:
Why was this removed? It didn't do the job?

Reply:
It did, but it gave a generic error like:
```
Sorry, your reply was invalid: value is too short. Min items is 1
```

Since there was already a check happening explicitly after that, I just used that so that we can give our own error

Reply:
Fair :+1: